doc/specs/#396 - Settings command.md

```diff
@@ -0,0 +1,175 @@
+---
+author: Ruben Guerrero @msftrubengu
+created on: 2020-06-03
+last updated: 2020-06-03
+issue id: 396
+---
+
+# Settings command
+
+For [#396](https://github.com/microsoft/winget-cli/issues/396)
+
+## Abstract
+
+The winget.exe client must support having a feature that allows users to specify settings. This will be in the form of an editable file. The file must be in a location where a user can easily get it and modify it.
+
+## Inspiration
+
+Add ability for user to set their own preferences.
+
+## Solution Design
+
+### Format
+
+The WinGet settings file needs to be in a readable format for users.
+
+#### Option 1: YAML 
+
+WinGet already knows how to handle YAML files via yaml-cpp. To follow the manifest style, the properties will be PascalCased.
+
+#### Option 2: JSON
```

Comment:
I think JSON is the right choice, as it is more natural here.  But we do need to determine how will yaml-cpp parses JSON with comments.

Reply:
Shall we use jsoncpp from the terminal PR that Ruben mentioned previously?

Reply:
_Another_ file format? Pick one, don't mix and match. That's confusing.

Reply:
@riverar, is that a vote for YAML, or a vote against a JSON with non "standard" comments.  I'm interpreting it as for YAML to be consistent with the manifests, but making sure.

Reply:
I'm voting for consistency with repository yaml just to keep the code base smaller, less risk of failure. Comments aren't important in my mind, I think docs/schema/ui can easily fill this gap.